### PR TITLE
Define API

### PR DESCRIPTION
**Pull request by Baschdl**
@juliusrueckin Do we need more routes? What do you think about the current state?

### CODE REVIEW

**Comment by Baschdl** [comment]
Do we request the node and edge types from neo4j or should our API provide a corresponding route?

File: api/server.py [hunk 1]
```diff
@@ -7,6 +7,43 @@ def run(port, hostname, debug_mode):
     app.run(host=hostname, port=port, debug=debug_mode)
 
 
-@app.route("/")
-def hello_world():
+# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
```
Comment (Baschdl) [inline api/server.py]: @KDD-OpenSource/computation This point concerns you. If you'll write this functionality in Java, we have to pass it to you in another way (maybe over some Cypher query).

File: api/server.py [hunk 2]
```diff
@@ -7,6 +7,43 @@ def run(port, hostname, debug_mode):
...
+        abort(400)
+
+
+@app.route("/node-sets", options="GET")
+def send_node_sets():
+    # TODO: Call fitting method in active_learning
+    return jsonify("Hello world")
+
+
+# TODO: If meta-paths for A and B will be written in Java, they will need this information in Java
```
Comment (Baschdl) [inline api/server.py]: @KDD-OpenSource/computation Same here.

**Comment by Baschdl** [comment]
I would recommend to merge this, even if we have a lot of TODOs in it. So everyone can add their functionality to it.

**Comment by Baschdl** [comment]
Merge #7 first

File: api/server.py (hunk 1 above)
Comment (feeds) [reply api/server.py]: @Baschdl maybe leave his for later?

**Pull request merged**